module/newrelic/README.md

```diff
@@ -0,0 +1,26 @@
+# Ergonode - New Relic
+
+## Documentation
+
+* Follow link to [Ergonode Documentation][docs],
+
+## Community
+
+* Get Ergonode support on **Stack Overflow**, [Slack][slack] or email team@ergonode.com
+* Follow us on [GitHub][github], [Twitter][twitter] and [Facebook][facebook],  
+
+## Contributing
+
+Ergonode is an Open Source. Join us as a [contributor][contribution]. 
+
+## About Us
+
+Ergonode development is sponsored by Bold Brand Commerce Sp. z o.o., lead by **Ergonode Core Team** and supported by Ergonode contributors. 
+
+[docs]: https://docs.ergonode.com
+[slack]: https://ergonode-community.slack.com
+[twitter]: https://twitter.com/ergonode
+[facebook]: https://www.facebook.com/ergonode
+[github]: https://github.com/ergonode
+[license]: ./LICENSE.txt
+[contribution]: http://docs.ergonode.com/#/community/contribution
```

Comment:
i think that be nice add hire same configuration informations.
eg. how work configuration from  module/newrelic/src/Application/DependencyInjection/Configuration.php